### ISSUE TITLE
Skyblock Generation Problem

### ISSUE THREAD

**Issue by bennytheg** [larryTheCoder/ASkyBlock#1]
Big problem, the Skyblock world generates with a 3 block water base for me, people can let their lava flow down and then it makes the water surface into block, game breaking in my opinion

**Comment by larryTheCoder** [comment]
There is a configuration for that, in `worlds.yml` you can specify `seaLevel` to 0, thus making sure the world you are using will not generate any water. But in this case, you need to reset the whole island if u set that sea level at the beginning, because the chunks are already loaded.

![image](https://user-images.githubusercontent.com/15889642/80521072-fbd33000-89bc-11ea-9a90-a74964cc67b3.png)

If you don't want to, try to use WorldEdit to clear up the water from those chunks.

##### tl;dr, default sea level are now 0 since 2c4e29f41940287f964ff1c64e571650ca2594eb
